playbook-website/app/javascript/site_styles/_kit_api.scss

```diff
@@ -0,0 +1,9 @@
+@import "playbook-ui/dist/tokens/typography";
+.kearning{
+  letter-spacing: $lspace_looser;
+}
+
+.card{
+  max-width: fit-content;
+  display: inline-block;
+}
```

Comment:
@co1in-1ewis @jasperfurniss Easy updates to our global props. There's not reason we should have to add this small CSS if it were available through GP.